### ISSUE TITLE
grapple beam oopsie

### ISSUE THREAD

**Issue by tomlube** [AxioDL/metaforce#360]
https://youtu.be/yTa9nhoQJw8

happens on most grapple points if you press A.

**Comment by Antidote** [comment]
I need more details, but I'm guessing you're not supposed to be able to fire the arm canon while using the grapple beam lol

**Comment by tomlube** [comment]
> I need more details, but I'm guessing you're not supposed to be able to fire the arm canon while using the grapple beam lol

Yeah that's correct sorry. Shooting while grappling is an Metroid Prime 2 thing

**Comment by tomlube** [comment]
Fixed in Reborn.
